### DISCUSSION TITLE
Flash/blockwise/ring attention on Turing GPU for Kaggle Competition

### DISCUSSION THREAD

**Discussion by defdet** [erfanzar/EasyDeL#137] [Q&A] [answered]
Hello, I was wondering, is it possible to run flash (or better) attention on Turing GPUs (like T4) with your framework? I would much rather use TPUs obviously, but Kaggle competitions allow only GPUs for submission and we really need large context. Pytorch currently doesn't have support for these GPUs.

**Comment by erfanzar** [comment, marked as answer]
Hello, I'm working on that right now but, yes soon I assume it will be possible

**Comment by defdet** [comment]
Looking forward to it, that would help a lot!
